core/src/main/java/com/chopshop166/chopshoplib/sensors/gyro/PigeonGyro2.java

```diff
@@ -0,0 +1,117 @@
+package com.chopshop166.chopshoplib.sensors.gyro;
+
+import com.ctre.phoenix.sensors.Pigeon2;
+
+import edu.wpi.first.util.sendable.SendableBuilder;
+
+/**
+ * Gyro Base wrapper for the Pigeon IMU
+ */
+public class PigeonGyro2 implements SmartGyro {
+
+    /** The wrapped object. */
+    final private Pigeon2 gyro;
+    /** Boolean to control inverted output. */
+    private boolean inverted;
+
+    /**
+     * Create the wrapper.
+     *
+     * @param gyro The object to wrap.
+     */
+    public PigeonGyro2(final Pigeon2 gyro) {
+        super();
+        this.gyro = gyro;
+
+        // Automatically invert if the gyro is upside-down
+        final short accelerometerXYZ[] = new short[3];
+        this.gyro.getBiasedAccelerometer(accelerometerXYZ);
+        if (accelerometerXYZ[2] < 0) {
+            this.inverted = true;
+        }
+    }
+
+    /**
+     * Create the wrapper.
+     *
+     * @param deviceNumber The CAN ID of the Pigeon 2.
+     */
+    public PigeonGyro2(final int deviceNumber) {
```

Comment:
Couldn't this just call the previous constructer and just create the Pygeon2 object in there